docs/backup-restore.md

```diff
@@ -35,26 +48,14 @@ Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getti
 
 **Cosmos DB**
 
-You must create a Cosmos DB account with a Continuous backup policy to create point-in-time restore (PITR).
-Please follow [the basic strategy](#basic-strategy-to-create-a-transactionally-consistent-backup) section to create a backup.
+You must create a Cosmos DB account with a Continuous backup policy to create point-in-time restore (PITR). Backups are created continuously after enabling this.
 
 **DynamoDB**
 
-You must create tables with point-in-time recovery (PITR) and auto-scaling in DynamoDB. Point-in-time recovery creates continuous backup and auto-scaling provides better performance. Scalar DB Schema Loader enables PITR and auto-scaling by default.
-Please follow [the basic strategy](#basic-strategy-to-create-a-transactionally-consistent-backup) section to create a backup.
-
-#### Basic strategy to create a transactionally-consistent backup
-
-One way to create a transactionally-consistent backup is to take a backup while Scalar DB cluster does not have outstanding transactions. 
-If an underlying database supports a point-in-time snapshot/backup mechanism, you can take a snapshot during the period.
-If an underlying database supports a point-in-time restore/recovery mechanism, you can set a restore point to a specific time (preferably the midtime) in the period since the system takes backups for each operation in such a case.
-
-To easily achieve transactionally-consistent backup for Scalar DB on a non-transactional database is to use the [Scalar DB server](https://github.com/scalar-labs/scalardb/tree/master/server) (which is implemented `scalar-admin` interface) or implement the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application.
-You can use the [Client-side tool](https://github.com/scalar-labs/scalar-admin/tree/scalar-admin-dockerfile#client-side-tool) to pause the application without losing ongoing transactions.
-
-Note that when you use a point-in-time-restore/recovery mechanism, it is recommended to minimize the clock drifts between nodes (`scalar-admin` interface implemented application nodes or `Scalar DB server` nodes that requests a pause) by using clock synchronization such as NTP.
-Otherwise, the time you get as a paused duration might be too different from the time in which the pause was actually conducted, which could restore to a point where ongoing transactions exist.
-Also, it is recommended to pause a long enough time (e.g., 10 seconds) and use the midtime of the paused duration as a restore point since clock synchronization cannot perfectly synchronize clocks between nodes.
+You must create tables with point-in-time recovery (PITR) in DynamoDB. Point-in-time recovery creates a continuous backup for a table.
```

Comment:
What do you mean `create tables with PITR`?
PITR is something for creating tables?